kyaml/yaml/alias.go

```diff
@@ -10,14 +10,21 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/internal/forked/github.com/go-yaml/yaml"
 )
 
-const CompactSequenceStyle = "compact"
-const WideSequenceStyle = "wide"
+const (
+	WideSeqIndent    SeqIndentType = "wide"
+	CompactSeqIndent SeqIndentType = "compact"
+	DefaultIndent                  = 2
+)
 
-const DefaultIndent = 2
-const DefaultSequenceStyle = CompactSequenceStyle
+// SeqIndentType holds the indentation style for sequence nodes
+type SeqIndentType string
```

Comment:
nit: SeqIndentStyle represent the behavior better. Also, preserve the older `CompactSequenceStyle`, `WideSequenceStyle` names to reduce the changes on public symbols.